src/plugins/summaryWidget/src/SummaryWidget.js

```diff
@@ -114,6 +116,8 @@ define([
      */
     SummaryWidget.prototype.addHyperlink = function (url, openNewTab) {
         if (url) {
+            const sanitizeUrl = urlSanitizeLib.sanitizeUrl;
+            url = sanitizeUrl(url);
             this.widgetButton.attr('href', url);

```

Comment:
I think directly pass to attr? 
`this.widgetButton.attr('href', urlSanitizeLib.sanitizeUrl(url))` 
or
```
const sanitizedUrl = urlSanitizeLib.sanitizeUrl(url);
this.widgetButton.attr('href', sanitizedUrl);
```